magellan/tests/test_attr_equiv_blocker.py

```diff
@@ -0,0 +1,230 @@
+import os
+from nose.tools import *
+import pandas as pd
+import unittest
+
+import magellan as mg
+
+p = mg.get_install_path()
+path_for_A = os.sep.join([p, 'datasets', 'table_A.csv'])
+path_for_B = os.sep.join([p, 'datasets', 'table_B.csv'])
+l_key = 'ID'
+r_key = 'ID'
+l_block_attr_1 = 'zipcode'
+l_block_attr_2 = 'birth_year'
+l_block_attr_3 = 'name'
+r_block_attr_1 = 'zipcode'
+r_block_attr_2 = 'birth_year'
+r_block_attr_3 = 'name'
+l_output_attrs = ['zipcode', 'birth_year']
+r_output_attrs = ['zipcode', 'birth_year']
+l_output_prefix = 'l_'
+r_output_prefix = 'r_'
+
+bogus_attr = 'bogus'
+block_attr_multi = ['zipcode', 'birth_year']
+
+
```

Comment:
@sanjibkd 

Can you please add testcases checking for invalid input type. For example the input A table is not a dataframe, the block attributes are not of type string, the prefixes and output attrs are not of type string.

You can do sthg like this for checking

    if not isinstance(A, pd.DataFrame):
        logger.error('Input left table is not of type pandas data frame')
        raise AssertionError('Input left table is not of type pandas data frame')

    if not isinstance(l_prefix, six.string_types):
        logger.error('l_prefix is not of type string')
        raise AssertionError('l_prefix is not of type string')